lib/transform/transformers.js

```diff
@@ -31,10 +31,7 @@ export function assets (asset) {
     asset.fields.file,
      (newFile, file, locale) => {
        newFile[locale] = omit(file, 'url', 'details')
```

Comment:
IMHO, blacklisting here is easier to deal with than picking since the list will be bigger